INSTALL.md

```diff
@@ -53,18 +53,53 @@ sudo apt install -y dangerzone
 
 ### Fedora
 
-Add our repository following [these instructions](https://packagecloud.io/firstlookmedia/code/install#manual-rpm), or by running this script:
+### 1. Add our repository
+
+Start by installing our repository. This allows you to
+
+Open a terminal window and type the following commands:
 
 ```
-curl -s https://packagecloud.io/install/repositories/firstlookmedia/code/script.rpm.sh | sudo bash
+gpg --keyserver hkps://keys.openpgp.org --recv-key "DE28 AB24 1FA4 8260 FAC9  B8BA A7C9 B385 2260 4281"
+gpg --export --armor A7C9B38522604281 | sudo tee /etc/pki/rpm-gpg/RPM-GPG-dangerzone.pub > /dev/null
+sudo yum-config-manager --add-repo=https://packages.freedom.press/yum-tools-prod/dangerzone/dangerzone.repo
 ```
 
-Install Dangerzone:
 
+### 2. Install Dangerzone
+
+
+```
+sudo dnf install dangerzone
 ```
-sudo dnf install -y dangerzone
+
+<details>
+<summary>Importing GPG key 0x22604281: ... Is this ok [y/N]:</summary>
+
+After some minutes of running the above command (depending on your internet speed) you'll be asked to confirm the fingerprint of our signing key. This is to make sure that in the case our servers are compromized your computer stays safe. It should look like this:
+
+```console
+--------------------------------------------------------------------------------
+Total                                           389 kB/s | 732 MB     32:07
+Dangerzone repository                           3.8 MB/s | 3.8 kB     00:00
+Importing GPG key 0x22604281:
+ Userid     : "Dangerzone Release Key <dangerzone-release-key@freedom.press>"
+ Fingerprint: DE28 AB24 1FA4 8260 FAC9 B8BA A7C9 B385 2260 4281
+ From       : /etc/pki/rpm-gpg/RPM-GPG-dangerzone.pub
+Is this ok [y/N]:
 ```
 
+> **Note**: If it does not show this fingerprint confirmation or the fingerprint does not match, it is possible that our servers were compromized. Be distrustful and reach our to us.
```

Comment:
[nit, sorry] -- reach _out_, rather than 'our'

Reply:
Whoops. Thanks for taking a look and spotting this one!